app/routes/upload.step3.tsx

```diff
@@ -0,0 +1,5 @@
+const step3 = () => {
+  return <div>ImproveResume</div>
+}
```

Comment:
**nitpick**: The component currently only returns a placeholder. Ensure this is completed before deployment.